reference/conanfile/methods.rst

```diff
@@ -1544,3 +1544,25 @@ These are all the fields that can be adjusted, both in ``self.patterns.source``
 +--------------------------------------+---------------------------------------------------------------------------------------------------------+
 | framework                            | Patterns of the files from the folders: ``self.cpp.xxx.frameworkdirs``                                  |
 +--------------------------------------+---------------------------------------------------------------------------------------------------------+
+
+
+test()
+------
+
+The ``test()`` method is only used for ``test_package/conanfile.py`` recipes. It will execute immediately after ``build()`` has been called, and its goal is to
```

Comment:
```suggestion
The ``test()`` method is only used for *test_package/conanfile.py* recipes. It will execute immediately after ``build()`` has been called, and its goal is to
```